openslides_backend/action/actions/projector/project.py

```diff
@@ -0,0 +1,119 @@
+from typing import Any, Dict
+
+from ....models.models import Projection, Projector
+from ....shared.filters import And, FilterOperator, Not
+from ....shared.patterns import Collection, string_to_fqid
+from ....shared.schema import required_id_schema
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData
+from ..projection.create import ProjectionCreate
+from ..projection.update import ProjectionUpdate
+
+
+@register_action("projector.project")
+class ProjectorProject(UpdateAction):
+    """
+    Action to projector project.
+    """
+
+    model = Projector()
+    schema = DefaultSchema(Projection()).get_default_schema(
+        required_properties=["content_object_id"],
+        optional_properties=["options", "stable", "type"],
+        additional_required_fields={
+            "ids": {"type": "array", "items": required_id_schema}
```

Comment:
Uniqueness would be good.

Reply:
Add a check for uniqueness of the ids. Add a test.